### PR TITLE
Resolve markdown-link-check forbidden with renovate app link

### PR DESCRIPTION
**Pull request by MikeMcC399**
This PR resolves a new issue that `npm run check:markdown` fails on GitHub when checking the link https://renovateapp.com/ in the [README](https://github.com/cypress-io/github-action/blob/master/README.md) file. The error is HTTP 403 forbidden.

When run locally there is no failure and https://renovateapp.com/ redirects to https://renovatebot.com/ and then to https://www.mend.io/free-developer-tools/renovate/.

I have replaced the link
https://renovateapp.com/
by
https://github.com/renovatebot
and this passes the `npm run check:markdown` check on GitHub.

## Problem analysis

CloudFront is blocking access to https://www.mend.io/free-developer-tools/renovate/ from GitHub by responding with 403 forbidden. A `cy.visit('https://www.mend.io/free-developer-tools/renovate/)` run on GitHub also fails.

`curl -v https://www.mend.io/free-developer-tools/renovate/ -o /dev/null` run on GitHub shows:
```
< HTTP/2 403 
< server: CloudFront
< content-type: text/html
< x-cache: Error from cloudfront
```

### CODE REVIEW

**Comment by MikeMcC399** [comment]
This is a high priority item since it will block the release of any new version through [main.yml](https://github.com/cypress-io/github-action/blob/master/.github/workflows/main.yml) until it is merged.

**Comment by MikeMcC399** [comment]
@jaffrepaul 

Please merge this PR as the markdown issue is blocking all other PRs from successful testing.

**Review by jaffrepaul — APPROVED** [review]

**Pull request merged**
